lavanet/src/Makefile

```diff
@@ -0,0 +1,15 @@
+CC=gcc
+
+# original version was not showing in xscreensaver little window;
+# it also did not respond to resize window events (for example when run from terminal).
+
+CFLAGS = -Wall -O2
+CLIBS = -lX11 -lm
+
+all : lavanet.c vroot.h
+		$(CC) $(CFLAGS) -o lava lavanet.c $(CLIBS)
```

Comment:
I would call the executable the same as the project, `lavanet`.

That's mainly to cut down on confusion, if everything always has the same name you never need to start mapping names in your head.

Reply:
No problem with that, I just like shorter names :)